lib/cdo/aws/rds.rb

```diff
@@ -0,0 +1,120 @@
+require_relative '../../../deployment'
+require 'aws-sdk-rds'
+
+module Cdo
+  class RDS
+    # Create an RDS Aurora MySQL cluster cloned from a specified source cluster.
+    # @param source_cluster_id [String] DB cluster id of the cluster to clone.  Defaults to current environment's cluster.
+    # @param clone_cluster_id [String] DB cluster id to assign to clone.  Defaults to source cluster id + "-clone"
+    # @param instance_type [String]
+    def self.clone_cluster(
```

Comment:
should this function returns something so following tasks know if cloning db succeeds or not?

Reply:
It will raise an error if it is not successful.  I can have it return the ARN of the cluster, if it succeeds.  Would that help?

Reply:
Sorry I missed your reply. Raising exception is sufficient for my use case.